.github/workflows/gh-pages.yml

```
@@ -62,6 +66,14 @@ jobs:
           destination_dir: master/reference
         if: github.ref == 'refs/heads/master'
 
+      - name: Deploy master std
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs/std/book
```

`publish_dir` is the source directory, so actually something like 
```suggestion
          publish_dir: ./sway-lib-std/out
```

_could_ work.

---

wouldn't we want this to be `./sway-lib-std/out/doc` since we don't want all of the out directory?